Wangscape/tilegen/partition/TilePartitionerNoise.cpp

```diff
@@ -43,6 +44,19 @@ noise::module::ModulePtr TilePartitionerNoise::makeCornerModule(const Corners& c
     combiner.setInputModuleSource(2, central.getOutputModule());
     if (mDebugOutput)
```

Comment:
Done. Debug output required a lot of persistent objects, so I've made a class to do it.